docs/howtos/runtime_sessions.rst

```diff
@@ -1,5 +1,45 @@
-Use Experiments with Runtime sessions
-=====================================
+Use Experiments with Runtime sessions and sampler
+=================================================
+
+Problem
+-------
+
+You want to run experiments with a custom `SamplerV2
+<https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.SamplerV2>`_ service. 
```

Comment:
I would reference the more general class:

```suggestion
You want to run experiments with a custom :class:`qiskit.primitves.SamplerV2` instance. 
```

Reply:
That would go to `basesamplerv2` but I can change it